Extension/src/LanguageServer/extension.ts

```diff
@@ -1379,3 +1381,66 @@ export async function getIncludes(maxDepth: number): Promise<any> {
     const includes = await clients.ActiveClient.getIncludes(maxDepth);
     return includes;
 }
+
+async function onOTFDocs(): Promise<void> {
+    const response = await clients.ActiveClient.getOTFDocsInfo();
+
+    // Ensure the content is valid before proceeding.
+    const request = response.content;
+
+    if (request.length === 0) {
+        return;
+    }
+
+    const locale = getLocaleId();
+
+    const messages = [
+        vscode.LanguageModelChatMessage
+            .User(request + locale)];
+
+    const [model] = await vscode.lm.selectChatModels({
+        vendor: 'copilot',
+        family: 'gpt-4'
```

Comment:
Do you know how restrictive this ends up being? Do we need a way to select a fallback model in case, for example, VS Code updates the list of supported models and we don't react in time?

Reply:
I believe this is the most generic way to get the handle to gpt-4 in copilot. This is the same way as it's done in the sample from Isidor. If this ends up getting a lot of usage, it looks like gpt-3.5 and gpt-4o are also available. I think within these bundles they can update the model underneath without breaking compat.